integration/rust_test.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package integration
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/hex"
+	"os/exec"
+	"regexp"
+	"testing"
+
+	"github.com/hyperledger/aries-bbs-go/bbs"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRustSignerGoVerifier(t *testing.T) {
+	re := regexp.MustCompile(`^PK: ([0-9a-f]+)\nsig: ([0-9a-f]+)\n$`)
+
+	output, err := exec.Command("./rust_signer/target/debug/rust_signer").Output()
+	require.NoError(t, err)
+
+	sm := re.FindStringSubmatch(string(output))
+	require.Len(t, sm, 3)
+
+	pkBytes, err := hex.DecodeString(string(sm[1]))
+	require.NoError(t, err)
+
+	sigBytes, err := hex.DecodeString(string(sm[2]))
+	require.NoError(t, err)
+
+	messagesBytes := [][]byte{
```

Comment:
maybe export to a constant / global variable?

Reply:
Good point, done, and added comment that it needs to be equal to its rust counterpart